### PR TITLE
Add template-service-wizard to QtCreator

### PR DESCRIPTION
**Pull request by DunderRoffe**
Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

**Review by GordanM — APPROVED** [review]
LGTM

File: sde-cookbook/qtcreator/add-template-service-wizard.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+
+WIZARD_REPO_NAME="template-service-wizard"
+WIZARD_REPO_REVISION="c998e37203b663b352c2acd06c2255fd50a85ca1"
+
+QTCREATOR_PATH=$1
```
Comment (tobsan) [inline sde-cookbook/qtcreator/add-template-service-wizard.sh]: Consider quoting `$1` here

**Review by tobsan — APPROVED** [review]
Just had one remark.

**Comment by DunderRoffe** [comment]
Fixed @tobsan

**Pull request merged**